src/perf/bin/appmain.cpp

```diff
@@ -0,0 +1,204 @@
+/*++
+
+    Copyright (c) Microsoft Corporation.
+    Licensed under the MIT License.
+
+Abstract:
+
+    QUIC Perf Main execution runner.
+
+--*/
+
+#ifdef QUIC_CLOG
+#include "appmain.cpp.clog.h"
+#endif
+
+#define QUIC_TEST_APIS 1
+#include "quic_driver_main.h"
+#include "PerfHelpers.h"
+#include <quic_trace.h>
+
+#ifdef _WIN32
+
+//
+// Name of the driver service for quicperf.sys.
+// Must be defined before quic_driver_helpers.h is included
```

Comment:
I know it's a bigger change, but I still greatly dislike reliance on global #defines or variables. Pass things as input please. It makes the code much more readable.

Reply:
This ones actually for a different reason. Because its a macro, I can concatinate it with other macros to get the other versions of it I need later on. Without this, the code in driver helpers actually becomes a lot more complicated, as it will then have to manually concatinate strings in memory. I can do that, but that will also affect BVTs

Reply:
Ok.